python_modules/libraries/dagster-shared/dagster_shared/utils/config.py

```diff
@@ -0,0 +1,46 @@
+import os
+import sys
+from pathlib import Path
+from typing import Any
+
+import tomlkit
+import yaml
+
+
+def is_windows() -> bool:
+    return sys.platform == "win32"
+
+
+def _get_default_dg_cli_folder() -> Path:
+    if is_windows():
+        return Path.home() / "AppData" / "dg"
+    else:
+        return Path.home() / ".dg"
```

Comment:
I don't think we should use this because the cache doesn't live there on unix (currently anyway)-- can we name the file `dg.toml` and put it directly in the home directory or in `AppData\dg` for windows?